util/dynamicConfig.js

```diff
@@ -1,24 +1,38 @@
+/* eslint-disable no-param-reassign */
 const get = require("get-value");
 const unset = require("unset-value");
 
+function recurse(event, value, Config, field) {
```

Comment:
```suggestion
function setDynamicConfigValue(event, value, config, field) {
```
This name looks more appropriate rather than `recurse` & also don't use first letter upper-cased character in method args

Reply:
Updated.